packages/custom-webpack/package.json

```diff
@@ -42,6 +42,7 @@
     "@angular-devkit/core": "^11.0.0",
     "lodash": "^4.17.15",
     "ts-node": "^9.0.0",
+    "tsconfig-paths": "^3.9.0",
```

Comment:
I am currently merging part of your changes into my PR (#824) and adding `tsconfig-paths` was not needed. Why are you adding it, are there any benefits?

Reply:
As far as I remember `ts-node` does not support `paths` property, so if you want to have the exact same config (including paths) for your webpack (or index) configuration you need this one.

Reply:
Makes sense! But `ts-node` isn't needed for any Angular project since it compiles through `tsc` and does not require to be "started" through Node, so I wouldn't add this or do I miss any usecase here?

Reply:
`ts-node` is indeed not needed for Angular app, because Angular compile (not exactly `tsc`) builds it, that's correct.  
However, index transform and webpack config are not part of the app, they are part of the build chain, which in our case is `custom-webpack` builder. The builders needs `ts-node` in order to understand the TS syntax of the files it loads (webpack config and index transform).